core/cas-server-core-services/src/test/java/org/apereo/cas/authentication/DefaultMultifactorTriggerSelectionStrategyTests.java

```diff
@@ -152,7 +153,7 @@ private static HttpServletRequest mockRequest(final String provider) {
 
     private static RegexRegisteredService mockService(final String... providers) {
         final DefaultRegisteredServiceMultifactorPolicy policy = new DefaultRegisteredServiceMultifactorPolicy();
-        policy.setMultifactorAuthenticationProviders(ImmutableSet.copyOf(providers));
+        policy.setMultifactorAuthenticationProviders(new HashSet<>(Arrays.asList(providers)));
```

Comment:
This is probably more like `Stream.of(providers).collect(Collectors.toSet())`? Or should be?